mpact_bot.py

```diff
@@ -131,14 +131,14 @@ async def incoming_message_handler(event):
                 # creating records in the UserChatUnread model for maintaining
                 # unread count for each user and individual chat.
                 for user in User.objects.all():
-                    UserChatUnread.objects.create(
+                    user_chat_unread, _ = UserChatUnread.objects.get_or_create(
                         user_id=user.pk, room_id=event.message.peer_id.user_id
                     )
 
-            await save_send_message(msg_data, channel_layer)
+                await start_handler(event, channel_layer, msg_data)
```

Comment:
Does this mean that users don't need to issue any command to engage the bot, or is there a Telegram default command that this is falling back to?